inference/models/transformers/transformers.py

```diff
@@ -179,6 +180,18 @@ def download_model_artifacts_from_roboflow_api(self) -> None:
                 file=filename,
                 model_id=self.endpoint,
             )
+            if filename.endswith("tar.gz"):
+                subprocess.run(
```

Comment:
use subprocess instead of python's builtin tar.gz stuff bc that stuff is _very_ slow

Reply:
getting [`CalledProcessError`](https://docs.python.org/3/library/subprocess.html#subprocess.CalledProcessError) without any context may be strange, please find a way to communicate that better to the caller